src/references/game_events.md

```diff
@@ -366,7 +373,7 @@ function CheckEvent()
             eventObject = World.SpawnAsset(TEMPLATE, {parent = PARENT_OBJECT})
             templateSpawned = true
         end
-        if templateSpawned and not eventActive then
+        elseif templateSpawned and not eventActive then
```

Comment:
This will error. The `end` above would need to be remove for the `elseif` to be valid.